integration/nwo/commands/peer.go

```diff
@@ -149,15 +151,10 @@ func (c ChannelFetch) SessionName() string {
 func (c ChannelFetch) Args() []string {
 	args := []string{
 		"channel", "fetch", c.Block,
-	}
-	if c.ChannelID != "" {
-		args = append(args, "--channelID", c.ChannelID)
-	}
-	if c.Orderer != "" {
-		args = append(args, "--orderer", c.Orderer)
-	}
-	if c.OutputFile != "" {
-		args = append(args, c.OutputFile)
+		"--channelID", c.ChannelID,
+		"--orderer", c.Orderer,
```

Comment:
Do we never use this command to pull blocks from a peer? Certainly channelID should not be empty, but it seems that the others could be?

Reply:
I made this change because the code here was unnecessarily complex and inconsistent with (at least the majority) of the command helpers. For these args, we can just set them regardless of whether they're empty or not. The CLI logic will fetch from the peer if the specified orderer arg is empty.